node/compile-hook.js

```diff
@@ -80,6 +83,7 @@ createWrapperManager(exts, ".mjs");
 addWrapper(Mp._compile, function(func, content, filename) {
   const options = Object.assign({ filename }, compileOptions);
   options.cacheKey = () => getCacheKey(filename);
+  runtime.enable(this);
```

Comment:
This is just for adding the runtime helpers. This doesn't opt-in arbitrary code into the reify compiler.